scripts/deploy.py

```diff
@@ -145,9 +145,7 @@ def update_doi(self, doi):
             draft = target_deposit
         else:
             # found the existing deposit - so let's make a new version.
-            response = self.post(
-                "%s/actions/newversion" % target_deposit["links"]["self"]
-            )
+            response = self.post(target_deposit["links"]["newversion"])
```

Comment:
This change came from the original error. If I add a debug log line of 

```python
print("request url was %s. target_deposit['links']['newversion'] is %s" % (url, target_deposit["links"]["newversion"]))
```
This is what I get in the output. Not quite sure why they changed it, that's OSS life I guess.
![image](https://github.com/rseng/zenodo-release/assets/4874123/b64c9ea1-126b-4972-8342-b0244dfe16e3)